dronegen/darwin_pipelines.go

```diff
@@ -0,0 +1,532 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+const boringCryptoBranch = "dev.boringcrypto.go1.16"
```

Comment:
I would be tempted to put this in common.go next to the `goRuntime` constant so it's obvious that both need to be updated for any future Go version changes.